src/init.cpp

```diff
@@ -1157,11 +1157,10 @@ bool CheckHostPortOptions(const ArgsManager& args) {
         "-port",
         "-rpcport",
     }) {
-        if (args.IsArgSet(port_option)) {
-            const std::string port = args.GetArg(port_option, "");
+        if (const auto port{args.GetArg(port_option)}) {
```

Comment:
For all `GetArg` calls in this commit. Perhaps I am confused but should be const ref? Would also be more clear to continue to specify the type here.

```suggestion
        if (const std::string& port{args.GetArg(port_option)}) {
```